tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Builders/OrderBuilder.cs

```diff
@@ -15,27 +15,15 @@ private OrderBuilder()
                 OrganisationId = Guid.NewGuid(),
                 OrganisationName = "Organisation Name",
                 OrganisationOdsCode = "Ods Code",
-                OrganisationAddress = new Address { AddressId = 1, Line1 = "Line1", Line2 = "Line2", Line3 = "Line3", Town = "Leeds",County = "W Yorks", Postcode = "LS1 123", Country = "W YORKS"},
-                OrganisationContact = new Contact { ContactId = 1, FirstName = "Fred", LastName = "Robinson", Email = "f@emai.com", Phone = "12345678912"},
+                OrganisationAddress = AddressBuilder.Create().WithLine1("1 Some Ordering Party").Build(),
                 Created = DateTime.UtcNow,
                 LastUpdated = DateTime.UtcNow,
                 LastUpdatedBy = Guid.NewGuid(),
                 LastUpdatedByName = "Bob Smith",
                 OrderStatus = new OrderStatus() { OrderStatusId = 1, Name = "Submitted" },
                 SupplierId = "Some supplier id",
                 SupplierName = "Some supplier name",
-                SupplierAddress = new Address
-                {
-                    Line1 = "Some address line one",
-                    Postcode = "LS1 1SX"
-                },
-                SupplierContact = new Contact
-                {
-                    FirstName = "Sam",
-                    LastName = "Smith",
-                    Email = "sam.smith@email.com",
-                    Phone = "0123456789"
-                }
+                SupplierAddress = AddressBuilder.Create().WithLine1("1 Some Supplier").Build()

```

Comment:
Do you not want to keep the supplierContact in? Is there a reason you have removed it and not put anything else in its place?

Reply:
After our phone call it makes sense 😄